api/envoy/config/filter/http/rbac/v2/rbac.proto

```diff
@@ -23,7 +23,7 @@ message RBAC {
   // Specify the enforced RBAC rules to be applied globally.
   // Filter's decision based on enforced rules has impact on user experience.
   // Disable the filter for enforcement mode if rules isn't set.
-  config.rbac.v2alpha.RBAC enforced_rules = 2;
+  config.rbac.v2alpha.RBAC rules = 2;
```

Comment:
please change back to 1 (and remove the reserved).

Reply:
done.